src/main.js

```diff
@@ -71,7 +76,7 @@ services.app.reAuthenticate().catch((err) => {
   } else {
     console.error(err);
   }
-}).finally(() => {
+}).then(() => {
```

Comment:
Why then instead of `finally`?

Actually since you touched this code there is one thing we can improve in it that I noticed yesterday:

If API is down when the app is opened in a browser window the code that request the version below will fail and rais an error which will propagate to the top and nothing will be rendered at all. We probably want to catch an error when the API is not available and either set the versions to `n/a` or display a page that says that the API is not available.

Reply:
Sorry! I put back the `finally`, then fix the "api not available" error in another PR. Thanks!